src/Core/Utils.hh

```diff
@@ -23,6 +23,9 @@
   }
 #endif
 
+#define MAX_FRAME_RATE 1.0f / 60.0f
```

Comment:
Framerate is associated with number of frames per second. Maybe rename to MIN_FRAME_TIME?